### PR TITLE
Columns toggler tooltip

### PR DESCRIPTION
**Pull request by smogg**

### CODE REVIEW

File: src/desdemona/ui/table.cljs [hunk 1]
```diff
@@ -2,7 +2,7 @@
   (:require [wilson.dom :as wd]
             [reagent.session :as session]
             [dommy.core :as d :refer-macros [sel sel1]]
-            [desdemona.ui.dom :refer [get-el-height get-el-width]]
+            [desdemona.ui.dom :as dd :refer [get-el-height get-el-width]]
```
Comment (TimSimpsonR) [inline src/desdemona/ui/table.cljs]: Is get-el-height and get-el-width being used dynamically somehow?

File: src/desdemona/ui/dom.cljs [hunk 2]
```diff
@@ -12,6 +12,12 @@
 (def menu-item
   (r/adapt-react-class (aget js/ReactBootstrap "MenuItem")))
 
+(def overlay-trigger
+  (r/adapt-react-class (aget js/ReactBootstrap "OverlayTrigger")))
+
+(def tooltip
+  (r/adapt-react-class (aget js/ReactBootstrap "Tooltip")))
```
Comment (TimSimpsonR) [inline src/desdemona/ui/dom.cljs]: You could make a function to create these, like "(defn react-class [bootstrap-name] (r/adapt-react-class (aget js/ReactBootstrap bootstrap-name))", and then define them a bit simpler with "(def button (react-class "Button"))". Though to be fair this code is already pretty simple right now.

**Comment by TimSimpsonR** [comment]
Aside from the questions about unused vars, looks good to me.

File: src/desdemona/ui/table.cljs (hunk 1 above)
Comment (smogg) [inline src/desdemona/ui/table.cljs]: They are used in multiple places. I guess the `:refer` is not necessary, the names was just long enough for my taste

File: src/desdemona/ui/dom.cljs (hunk 2 above)
Comment (smogg) [inline src/desdemona/ui/dom.cljs]: Good point, I'll probably make that happen once we get more imports like these

**Comment by smogg** [comment]
@TimSimpsonR unused vars?

**Comment by ehashman** [comment]
This looks fine to me.

**Pull request merged**